config/initializers/omniauth.rb

```diff
@@ -0,0 +1,12 @@
+Rails.application.config.middleware.use OmniAuth::Builder do
+  OmniAuth.config.on_failure = SessionsController.action(:destroy)
+  provider :facebook, 
+            ENV["facebook_app_id"], 
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Trailing whitespace detected.